### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by sgtphips** [sgtphips/codelab-css-intro#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](half-deer.surge.sh) and meets the following [project](http://lansingcodelab.com/lessons/css-intro/1) criteria:

- [x] Include at least one instance of the `margin` property.
- [x] Include at least one instance of the `padding` property.
- [x] Include at least one instance of the `border` property.
- [x] Include at least one instance of the `color` property.
- [x] Include at least one instance of the `background` property.
- [x] Include at least one instance of the `font-size` property.
- [x] Include at least one instance of the `line-height` property.
- [x] All properties must have a visible effect.
- [x] Return no errors (warnings are OK) when the HTML document is uploaded to the [W3C Markup Validation Service](http://validator.w3.org/#validate_by_upload)
- [x] Return no errors or warnings when contents of your CSS are pasted into [CSS Lint](http://csslint.net/)

**Comment by chrisvfritz** [comment]
Very well done! I can't stress enough how great it is to see very disciplined indentation and formatting in all your lessons. :smiley: That's something it takes most students a while to learn. There's nothing _wrong_ here, but I do see a couple areas for possible improvement.

### `line-height` with unitless values

This isn't technically _wrong_, but while `line-height` accepts values with various units, the most common format (and the one I recommend) is unitless. For example, `line-height: 1` will give you the equivalent of single spacing in a program like Microsoft Word. `line-height: 2` will give you double spacing, etc. For most websites, I personally find myself using something between `1.4` and 1.6` for optimally readable text.

### More specific HTML5 elements

One other thing that's outside of the scope of this lesson, but I'll share with you since you're already relatively advanced, is there are other elements I would use in place of where you're using `section` elements. The rule for when to use them is admittedly vague, but the Mozilla Developer Network (which is an excellent resource for readable HTML element documentation), uses [this definition](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section):

> The HTML `<section>` element represents a generic section of a document, i.e., a thematic grouping of content, typically with a heading. Each `<section>` should be identified, typically by including a heading (`<h1>`-`<h6>` element) as a child of the `<section>` element.

So I might use that for different sections within an `<article>`, for example. If you don't have a heading for each section, that's usually an indication that they aren't really appropriate here. In your case though, there are other, more specific elements, that I would use instead:

- instead of `<section id="header">`, use [the `<header>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)
- instead of `<section id="body">`, use [the `<main>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main)
- instead of `<section id="aside-main">`, use [the `<aside>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main)
- instead of `<section id="footer">`, use [the `<footer>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer)
- it may also be good to wrap [your `<ul id="top-nav">`](https://github.com/sgtphips/codelab-css-intro/blob/master/index.html#L10-L14) in [a `<nav>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)

Let me know if that makes sense to you and if you have any questions/comments!

**Comment by sgtphips** [comment]
Yes this all makes sense to me. Should I redo it to get the squirrel?

**Comment by chrisvfritz** [comment]
Yes, let's make those changes for the practice and to ensure a great-looking portfolio on GitHub.

**Comment by sgtphips** [comment]
@chrisvfritz Can this be marked complete please

**Comment by chrisvfritz** [comment]
It looks like there's still [a leftover `em` value for `line-height`](https://github.com/sgtphips/codelab-css-intro/blob/master/style.css#L55) and though `index.html` has been updated with more semantic elements, the CSS selectors in `style.css` are still referring to the old elements, breaking many of the styles.
